lib/msf/core/exception.rb

```diff
@@ -21,7 +20,7 @@ module Exception
 # can be obtained through the options attribute.
 #
 ###
-class OptionValidateError < ArgumentError
+class Msf::OptionValidateError < ArgumentError
```

Comment:
Just confirming if this is needed?

```suggestion
class OptionValidateError < ArgumentError
```